_posts/0000-01-02-elz-harri.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Everything is so funny. Ha ha ha ha.
![Bullyland-Pteranodon-model](https://user-images.githubusercontent.com/69866210/90689234-b70a2800-e23d-11ea-92a3-21a06c9fe644.jpg)